### PR TITLE
Add basic usage testcase

### LINKED ISSUE #1: Add example usage

### LINKED ISSUE #6: Add tests

### PR DESCRIPTION
**Pull request by theilgaard**
Closes #1

### CODE REVIEW

**Comment by hoeg** [comment]
I think this PR is more related to https://github.com/hoeg/PadDown/issues/1 . This is a great example of how to use PadDown.

For unittesting I had imagined tests of plaintexts of size: 1, BLOCKSIZE, BLOCKSIZE+1 and a longer plaintext, no need to use AES or some other cipher, the `hasValidPadding` function could just return some predefined values.

Also we should add a task to the `.circle/config.yml` file to make sure the tests are run on CircleCi.

Consider if we should have a test for a PadChecker this is not implemented correctly. How do handle failure to decrypt in this case?

**Review by hoeg — CHANGES_REQUESTED** [review]
See comment on PR

**Review by hoeg — APPROVED** [review]

**Comment by theilgaard** [comment]
Don't merge this yet, the tests are failing. I'll look at it during the week.

**Comment by hoeg** [comment]
Lets get this added to CI. I'll suggest that we use github for this and move away from the circle ci that has been started. Thoughts?

**Comment by theilgaard** [comment]
Github Actions is definitely useable, however it has poor support for removing artifacts. At least I haven't been able to do it, thus building docker images sucks. 

Perhaps docker is also overkill for now? We could just activate a venv, build and pytest.

**Comment by theilgaard** [comment]
By not removing artifacts, we run out of space and money on github fast :)

**Comment by hoeg** [comment]
I agree that docker is overkill for now. My stand is that the artifact from a release is the code with a requirement.txt file such that users can incorporate the project into their own containers. Anyhow, we use what is easiest for us and provides us with the best feedback 🤷‍♂️

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+pycryptodome==3.9.0
```
Comment (hoeg) [inline requirements.txt:1]: Should structlog be added here?

Comment (theilgaard) [reply requirements.txt:1]: It should yes, please disregard this pull request, I'm just working on the branch.

**Pull request closed without merging**